packages/core/test-utils/src/utils.js

```diff
@@ -588,14 +589,51 @@ function prepareBrowserContext(
         return null;
       },
     },
-    currentScript: {
-      src: 'http://localhost/script.js',
-    },
+
+    currentScript: null,
   };
 
   var exports = {};
+
+  function PatchedError(message) {
+    const patchedError = new Error(message);
+    const stackStart = patchedError.stack.indexOf('at new Error');
+    const stackEnd = patchedError.stack.includes('at Script.runInContext')
+      ? patchedError.stack.indexOf('at Script.runInContext')
+      : patchedError.stack.indexOf('at runNextTicks');
+    const stack = patchedError.stack.slice(stackStart, stackEnd).split('\n');
+    stack.shift();
+    stack.pop();
+    stack.unshift(`    at getBundleURL (${bundle.name})`);
```

Comment:
So now every stack traces will contain this `getBundleURL` entry, even if the error was thrown somewhere else?

Reply:
perhaps we could change it to `    at (${bundle.name})`?

Reply:
The problem isn't the function name, it's that we're effectively hardcoding where the function is defined, even if it's defined in another bundle.